### ISSUE TITLE
chorus esp and self anvil look familiar, I wonder who wrote them

### ISSUE THREAD

**Issue by bush-did-711** [Simple-Github-ORG/OctoHack-SRC#2]
🤔

**Comment by M3gaPixel** [comment]
hm

**Comment by notperry12345678902** [comment]
fr fr

**Comment by ssllllll** [comment]
yes but primoctopus made a list specifying where did he jew those modules

https://github.com/Primooctopus33/octohack

**Comment by bush-did-711** [comment]
true that's pretty based

**Comment by master7720** [comment]
true true

**Comment by notperry12345678902** [comment]
> true that's pretty based

Not really he was forced to make it since JJBrother when making a fake leak literally leaked the real one (Cause anarchy ppl are that dumb) and I spoke to him on Discord about all the stuff that was skidded (he never admitted to aswell) until it just now got leaked so now he admits to all this so I personally say it's not based.

**Comment by bush-did-711** [comment]
this says a lot about society
